tests/components/derivative/test_sensor.py

```diff
@@ -89,30 +90,46 @@
 async def test_dataSet2(hass):
     """Test derivative sensor state."""
     await setup_tests(
-        hass, {"unit_time": "s"}, times=[20, 30], values=[5, 0], expected_state=-0.5
+        hass,
+        {"unit_time": TIME_SECONDS},
+        times=[20, 30],
+        values=[5, 0],
+        expected_state=-0.5,
     )
 
 
 async def test_dataSet3(hass):
     """Test derivative sensor state."""
     state = await setup_tests(
-        hass, {"unit_time": "s"}, times=[20, 30], values=[5, 10], expected_state=0.5
+        hass,
+        {"unit_time": TIME_SECONDS},
+        times=[20, 30],
+        values=[5, 10],
+        expected_state=0.5,
     )
 
-    assert state.attributes.get("unit_of_measurement") == "/s"
+    assert state.attributes.get("unit_of_measurement") == f"/{TIME_SECONDS}"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5054188)